examples/stencil_computation/interpolate_stencil.hpp

```diff
@@ -32,7 +32,7 @@ struct interpolate_stencil {
   private:
     using p_in1 = gridtools::arg<0, data_store_t>;
     using p_in2 = gridtools::arg<1, data_store_t>;
-    using p_weight = gridtools::arg<2, gridtools::global_parameter<backend_t, double>>;
+    using p_weight = gridtools::arg<2, gridtools::global_parameter<double>>;
```

Comment:
Same here. Is there also API-compatibility for this case?

Reply:
yes